projects/igniteui-angular/src/lib/progressbar/linearbar.component.spec.ts

```diff
@@ -45,405 +44,259 @@ describe('IgxLinearBar', () => {
-
-            fixture.componentInstance.striped = true;
-            fixture.detectChanges();
-
-            expect(linearBar.classList.contains(STRIPED_CLASS)).toEqual(true);
-
-            fixture.componentInstance.type = 'success';
-            fixture.detectChanges();
+        progress.striped = true;
+        fixture.detectChanges();
 
-            expect(linearBar.classList.contains(SUCCESS_TYPE_CLASS)).toEqual(true);
-            expect(linearBar.classList.contains(STRIPED_CLASS)).toEqual(true);
-        });
+        expect(linerBar.classList.contains(STRIPED_CLASS)).toEqual(true);
+    });
 
-        it('Prevent constant update of progress value when value and max value differ', fakeAsync(() => {
-            const fix = TestBed.createComponent(InitLinearProgressBarComponent);
-            fix.detectChanges();
+    it('should stay striped when the type has changed', () => {
+        const linearBar = progress.element.nativeElement;
 
-            const bar = fix.componentInstance.linearBar;
-            const maxVal = 3.25;
-            const value = 2.55;
-            bar.step = 0.634;
-            bar.max = maxVal;
-            bar.value  = value;
+        progress.striped = true;
+        fixture.detectChanges();
 
-            tick(tickTime + tickTime); // enough time to exceed the progress update.
-            fix.detectChanges();
+        expect(linearBar.classList.contains(STRIPED_CLASS)).toEqual(true);
 
-            const progressBarContainer = fix.debugElement.nativeElement.querySelector(LINEAR_BAR_TAG);
-            expect(parseFloat(progressBarContainer.attributes['aria-valuenow'].textContent)).toBe(value);
-            expect(bar.value).toBe(value);
-        }));
+        progress.type = 'success';
+        fixture.detectChanges();
 
-        it('When enable indeterminate mode, then the appropriate class should be applied.', () => {
-            const fix = TestBed.createComponent(LinearBarComponent);
-            fix.detectChanges();
+        expect(linearBar.classList.contains(SUCCESS_TYPE_CLASS)).toEqual(true);
+        expect(linearBar.classList.contains(STRIPED_CLASS)).toEqual(true);
+    });
 
-            const bar = fix.debugElement.nativeElement.querySelector(LINEAR_BAR_TAG);
-            expect(bar.classList.contains(INDETERMINATE_CLASS)).toEqual(false);
+    it('When enable indeterminate mode, then the appropriate class should be applied.', () => {
+        const bar = progress.element.nativeElement;
+        expect(bar.classList.contains(INDETERMINATE_CLASS)).toEqual(false);
 
-            fix.componentInstance.progressbar.indeterminate = true;
+        progress.indeterminate = true;
+        fixture.detectChanges();
 
-            fix.detectChanges();
-            expect(bar.classList.contains(INDETERMINATE_CLASS)).toEqual(true);
-        });
+        expect(bar.classList.contains(INDETERMINATE_CLASS)).toEqual(true);
     });
 });
-
-@Component({ template: `<igx-linear-bar [animate]="false"></igx-linear-bar>` })
```

Comment:
@Aleksandyr Take a look at the component before - animations were still off, so nothing has changed here.

Reply:
You are right.
I saw that 2 months ago I have disabled all animations when they were improved and that has been merged without noting that.
![image](https://user-images.githubusercontent.com/7037740/152783112-cd084c64-f0c6-40a6-97fc-726e7b6a84e7.png)
We need to change that because we miss several important cases when progress is updated with animation.
Especially when setting `max` and `value` runtime.

Reply:
@Aleksandyr I actually tried to add a few tests with animations on, and since we use the native browser animations, it made the tests really unstable. I think this can be left for a more "UI test" scenario, while keeping the current test suite as unit tests only.

Reply:
I tried as well and it faced the same result as you. I am going to add an issue for UI animation tests